definitions/features/katello.rb

```diff
@@ -17,7 +17,8 @@ def current_version
 
   def services
     [
-      system_service('elasticsearch', 30)
+      system_service('qpidd', 10),
```

Comment:
I know that the `foreman-maintain service` command is affected by this - but is there anything else I should consider?

Reply:
The `qpidd` and `qdrouterd` services are mapped to `pulp2` feature as per current implementation: https://github.com/theforeman/foreman_maintain/blob/master/definitions/features/pulp2.rb#L15 
The `pulp2` feature is available when `katello` is less than 4.0. 
Considering the fact that foreman-maintain package is not tightly coupled with Foreman/Katello/Satellite releases we should avoid mapping same service(s) twice in different features(pulp2, katello). 
So I suggest to conditionally map `qpidd` service in Katello if `pulp2` feature is not available.

Reply:
@upadhyeammit updated!